OpenRA.Mods.Cnc/Traits/Disguise.cs

```diff
@@ -76,13 +78,21 @@ class DisguiseInfo : ITraitInfo
 		[Desc("The condition to grant to self while disguised.")]
 		public readonly string DisguisedCondition = null;
 
-		[Desc("Triggers which cause the actor to drop it's disguise. Possible values: None, Attack, Damaged.")]
+		// Added this for more control over when disguises break
+		[Desc("Events leading to the actor breaking Disguise. Possible values are: Attack, Move, Demolish, and Damaged.")]
```

Comment:
I've got an idea for None assuming it doesn't do it already. How about it block use of the other flags while there? :)